lib/completion.py

```diff
@@ -99,19 +98,11 @@ def _serialize_completions(self, completions, identifier=None):
         _completions = []
         for completion in completions:
             _completion = {
-                'text': '%s%s' % (completion.name[
-                                  :completion._like_name_length],
-                                  completion.complete),
                 'snippet': self._generate_snippet(completion),
-                'displayText': completion.name,
                 'type': self._get_definition_type(completion),
-                # TODO: try to understand return value
-                # 'leftLabel': '',
+                'description': self._generate_signature(completion),
                 'rightLabel': self._additional_info(completion),
             }
-            if self.show_doc_strings:
```

Comment:
I'm feeling terrible to remove docstrings from completions. I understand that luck of formatting makes them unreadable,  but hopefully somebody will fix this in autocomplete-plus.

You didn't actually removed the option from `main.coffee` – maybe let's leave it as is and expand signature with docstrings if option is selected?